nem.rs

```diff
@@ -76,10 +133,32 @@ fn mnemonic(v: &[String], existing: Vec<&String>) -> String {
 
 fn main() {
     let mut args: Vec<_> = env::args().collect();
-    let contents = fs::read_to_string(".nem.toml").expect("Couldn't open file.");
-    let mut nem_file: NemFile = toml::from_str(&contents).expect("Couldn't parse toml file");
-    nem_file.file_name = String::from(".nem.toml");
-    nem_file.sort();
+
+    let mut nem_files = NemFiles {
+        nem_files: Vec::new(),
+    };
+    let mut path_buf = env::current_dir().expect("Couldn't get CWD");
+    loop {
+        path_buf.push(".nem.toml");
+        let path = path_buf.as_path();
```

Comment:
I think you don't have to make a `Path` object from the `PathBuf`, most of the methods seem to accept `AsRef<Path>`, which a `PathBuf` implements.